pkg/api/client_runtime.go

```diff
@@ -87,3 +92,12 @@ func (r *CloudClientRuntime) getRuntime(op *runtime.ClientOperation) *runtimecli
 	}
 	return r.regionRuntime
 }
+
+func overrideJSONProducer(r *runtimeclient.Runtime, opID string) func() {
```

Comment:
can we just leave a comment here explaining why this is being done? Alternatively the comment could be on line 82

Reply:
yep, I'll do that